docker-compose-devel.yml

```diff
@@ -7,5 +7,11 @@ services:
     entrypoint: "bash"
     volumes:
       - .:/home/centos/
+    environment:
```

Comment:
I think this might be better as:
```yaml
environment:
  TEST_NGINX_REDIS_HOST: redis
```

Using hash instead of array.

Reply:
Both options should be equivalent. I've changed it because the hash version looks a bit nicer.